src/core/meshcop/dtls.cpp

```diff
@@ -99,7 +97,11 @@ ThreadError Dtls::Start(bool aClient, ReceiveHandler aReceiveHandler, SendHandle
     mbedtls_ssl_conf_ciphersuites(&mConf, ciphersuites);
     mbedtls_ssl_conf_export_keys_cb(&mConf, HandleMbedtlsExportKeys, this);
     mbedtls_ssl_conf_handshake_timeout(&mConf, 8000, 60000);
+#ifdef _KERNEL_MODE
+    mbedtls_ssl_conf_dbg(&mConf, HandleMbedtlsDebug, NULL);
```

Comment:
Actually, I'd argue that we always use either `NULL` or `this` since this is just the callback context. I'd prefer just `NULL` because we never actually need the context right now.